doc.ddoc

```diff
@@ -67,6 +19,7 @@ $(SCRIPTLOAD js/cssmenu.js)
 </html>
 _=
 
+ROOT = .
```

Comment:
The current approach has the advantage that `<a href="$(ROOT)">Index</a>` works properly, which allows linking to the main page without hard-coding the `http://dlang.org/` URL.